### PR TITLE
Fixed Month shown in Treatment Summary. Issue 1407

### PR DESCRIPTION
**Pull request by peeyush14goyal**
Issue #1407 
`getMonth()` returns value from 0-11 so added 1 to get month value.

### CODE REVIEW

File: src/Components/Facility/TreatmentSummary.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ const TreatmentSummary = (props: any) => {
             <h2 className="text-center text-lg">INTERIM TREATMENT SUMMARY</h2>
 
             <div className="text-right font-bold">
-              {date.getDate()}/{date.getMonth()}/{date.getFullYear()}
+              {date.getDate()}/{date.getMonth() + 1}/{date.getFullYear()}
```
Comment (bodhish) [inline src/Components/Facility/TreatmentSummary.tsx]: Import moment and format with its helper. 
```suggestion
moment(date).format("LL"))
```

File: src/Components/Users/UserProfile.tsx [hunk 2]
```diff
@@ -136,6 +136,12 @@ export default function UserProfile() {
           if (!states.form[field]) {
             errors[field] = "This field is required";
             invalidForm = true;
+          } else if (
+            states.form[field] <= 0 ||
+            !/^\d+$/.test(states.form[field])
+          ) {
+            errors[field] = "Age must be a number greater than 0";
+            invalidForm = true;
```
Comment (bodhish) [inline src/Components/Users/UserProfile.tsx]: @peeyush14goyal looks like unrelated changed 🤔

File: src/Components/Common/SlideOver.gen.tsx [hunk 3]
```diff
@@ -1,21 +1,22 @@
 /* TypeScript file generated from SlideOver.res by genType. */
 /* eslint-disable import/first */
 
-import * as React from "react";
+
+import * as React from 'react';
```
Comment (bodhish) [inline src/Components/Common/SlideOver.gen.tsx:5]: gitignore `*.gen.tsx`

**Review by bodhish — CHANGES_REQUESTED** [review]

File: src/Components/Users/UserProfile.tsx (hunk 2 above)
Comment (peeyush14goyal) [reply src/Components/Users/UserProfile.tsx]: Maybe I made a branch from profile-testing branch 😅

File: src/Components/Common/SlideOver.gen.tsx (hunk 3 above)
Comment (peeyush14goyal) [reply src/Components/Common/SlideOver.gen.tsx:5]: This is there in gitignore but seems like its not working @bodhish

Comment (peeyush14goyal) [reply src/Components/Common/SlideOver.gen.tsx:5]: I have merged develop as well

**Comment by samholics** [comment]
tested, verified

The month in treatment summary looks fine for now

image

File: src/Components/Facility/TreatmentSummary.tsx (hunk 1 above)
Comment (peeyush14goyal) [reply src/Components/Facility/TreatmentSummary.tsx]: @bodhish Changes done
